### PR TITLE
napi 1.3.4 (new formula)

### PR DESCRIPTION
**Pull request by mstrzele**
Napi Projekt Client and Subotage subtitle format converter both written in bash

### CODE REVIEW

File: Library/Formula/napi.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+class Napi < Formula
+  desc "Napi Projekt Client and Subotage subtitle format converter both written in bash"
```
Comment (bfontaine) [inline Library/Formula/napi.rb]: Could you shorten the description? We try to keep “<name>: <desc>” under 80 chars, it’s 85 here. You could e.g. remove “both written in bash”. Thanks!

**Comment by bfontaine** [comment]
[The test is failing](http://bot.brew.sh/job/Homebrew%20Pull%20Requests/26853/version=mavericks/testReport/junit/brew-test-bot/mavericks/test_napi/):

```
==> napi.sh *
#00:0001 ERROR -> nie wszystkie wymagane narzedzia sa dostepne
Error: napi: failed
Failed executing: napi.sh *
/usr/local/Library/Homebrew/formula.rb:873:in `block in system'
/usr/local/Library/Homebrew/formula.rb:830:in `open'
/usr/local/Library/Homebrew/formula.rb:830:in `system'
/usr/local/Library/Formula/napi.rb:29:in `block in <class:Napi>'
/usr/local/Library/Homebrew/formula.rb:773:in `block in run_test'
/usr/local/Library/Homebrew/extend/fileutils.rb:17:in `mktemp'
/usr/local/Library/Homebrew/formula.rb:769:in `run_test'
/usr/local/Library/Homebrew/test.rb:28:in `block in <main>'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/timeout.rb:66:in `timeout'
/usr/local/Library/Homebrew/test.rb:27:in `<main>'
```

File: Library/Formula/napi.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  homepage "https://github.com/dagon666/napi"
+  url "https://github.com/dagon666/napi/archive/v1.3.4.tar.gz"
+  sha256 "32b8ee39e00e9ed8cbbdb02bb683ccbfb02bd1f109b1a1e9f97e5badbf12c996"
+
+  depends_on "ffmpeg" => :optional
+  depends_on "media-info" => :optional
+  depends_on "mplayer" => :optional
+
+  patch do
```
Comment (DomT4) [inline Library/Formula/napi.rb]: Please add comments briefly explaining what these patches do, when we can expect to remove them, etc.

File: Library/Formula/napi.rb [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+  def install
+    bin.mkpath
+    share.mkpath
+
+    system "./install.sh", "--bindir", bin, "--shareddir", share
+  end
+
+  test do
+    system "napi.sh", "*"
```
Comment (DomT4) [inline Library/Formula/napi.rb]: `system bin/"napi.sh"`.

**Comment by mstrzele** [comment]
@DomT4 I applied the changes proposed by you, but I still have a problem with the @bfontaine comment. The only one dependency is required by `napi.sh`. You can install `ffmpeg` or `mplayer` to get the required functionality, but I don't know how to achieve this in Formula syntax (maybe the first dependency should be required?)

**Comment by mikemcquaid** [comment]
@mstrzele Yeh, one of them should be `:required`; probably `ffmpeg`?

**Comment by mstrzele** [comment]
I changed the `ffmpeg` to be required, but now I'm getting this error after the build:

```
==> Installing napi dependency: ffmpeg
Error: /usr/local/opt/pkg-config not present or broken
Please reinstall pkg-config. Sorry :(
```

Am I doing something wrong or this issue is related with `ffpmeg` directly?

File: Library/Formula/napi.rb [hunk 4]
```diff
@@ -0,0 +1,35 @@
+class Napi < Formula
+  desc "Napi Projekt Client and Subotage subtitle format converter"
+  homepage "https://github.com/dagon666/napi"
+  url "https://github.com/dagon666/napi/archive/v1.3.4.tar.gz"
+  sha256 "32b8ee39e00e9ed8cbbdb02bb683ccbfb02bd1f109b1a1e9f97e5badbf12c996"
+
+  depends_on "ffmpeg" => :required
```
Comment (DomT4) [inline Library/Formula/napi.rb]: This should just be `depends_on "ffmpeg"`. Should also solve your `pkg-config` failure message.

**Comment by mstrzele** [comment]
Thanks @DomT4!

**Comment by bfontaine** [comment]
Could one of @Homebrew/owners give their :+1: here?

**Comment by tdsmith** [comment]
I thought ffmpeg should be required or recommended?

File: Library/Formula/napi.rb [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    bin.mkpath
+    share.mkpath
+
+    system "./install.sh", "--bindir", bin, "--shareddir", share
+  end
+
+  test do
+    system bin/"napi.sh", "*"
+
+    assert_equal 0, $?.exitstatus
```
Comment (tdsmith) [inline Library/Formula/napi.rb]: `Formula#system` already does this, so you can skip it here

File: Library/Formula/napi.rb [hunk 6]
```diff
@@ -0,0 +1,37 @@
+class Napi < Formula
+  desc "Napi Projekt Client and Subotage subtitle format converter"
+  homepage "https://github.com/dagon666/napi"
+  url "https://github.com/dagon666/napi/archive/v1.3.4.tar.gz"
+  sha256 "32b8ee39e00e9ed8cbbdb02bb683ccbfb02bd1f109b1a1e9f97e5badbf12c996"
+
+  depends_on "wget"
```
Comment (mikemcquaid) [inline Library/Formula/napi.rb]: Is this needed at runtime? What for?

File: Library/Formula/napi.rb [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+  depends_on "p7zip" => :optional
+  depends_on "ffmpeg" => :optional
```
Comment (mikemcquaid) [inline Library/Formula/napi.rb]: As mentioned: this should be `recommended`, no?

**Comment by DomT4** [comment]
Tim & Mike's comments need addressing here, Thanks!

**Comment by DomT4** [comment]
Closing due to no response. Will welcome a new PR with the issues highlighted addressed. Thanks for your submission to Homebrew!

**Pull request closed without merging**